### ISSUE TITLE
Taker Fee calculation

### ISSUE THREAD

**Issue by zavan** [tugwitt/bitmex_utils#1]
Hey, good job getting the formulae. But I believe you made one interpretation error:

I think the "taker fee" variables in the formulae are supposed to be multiplied by the position value, not used as is (a percentage), this wouldn't make sense.

Example:

Entry Price: 8840.45
Size: 20000 contracts
Leverage: 5x

So, at entry:
1 contract = 1/8840.45 = 0.00011312 BTC
Position value = size * contract value = 20000 *  0.00011312 = 2.26232828 BTC
Taker Fee = 0.075% * position value = 0.075% * 2.26232828 = **0.00169675** BTC
Initial Margin = (1/5) - **0.00169675** - **0.00169675** = 0.19660651 BTC
Maintenance Margin = 0.5% * **0.00169675** = 0.00000848 BTC

I believe maybe this is what's causing your few dollars difference error.

**Comment by nottug** [comment]
Yeah it looks like you're right, appreciate it. Pull request accepted.
